hazelcast/test/src/logger_test.cpp

```diff
@@ -0,0 +1,130 @@
+#include <chrono>
```

Comment:
Can you put a test for default logger output format is correct?

Reply:
Done.